src/_base/harness/attributes/common.yml

```
@@ -231,15 +231,22 @@ attributes.default:
         sample_rate: 25
         service: web
     ext-xdebug:
+      version: "= (@('php.version') >= 8.0 ? '3' : '2')"
```

```suggestion
      version: "= (version_compare(@('php.version'), '8.0', '>=') ? '3' : '2')"
```
Based on similar feedback from Andy in https://github.com/inviqa/harness-base-php/pull/506